docs/installation-guide-azure.md

```diff
@@ -38,13 +38,20 @@ az storage account keys list -g ${NAME}-terraform  --account-name ${NAME}terrafo
 
 ### Create service principal
 
-To get subscription use `az account list | jq '.[].id'`
+Create resource group for cluster. We need one to assign permissions.
 
 ```
-az ad sp create-for-rbac --name=${NAME}-sp --role="Contributor" --scopes="/subscriptions/XXX..."
+az group create -n ${NAME} -l westeurope
 ```
 
-Make sure to save output, it will be needed in next step.
+Create service principal with permissions limited to resource group.
+
+```
+export SUBSCRIPTION_ID=$(az account list | jq '.[].id')
+az ad sp create-for-rbac --name=${NAME}-sp --role="Contributor" --scopes="/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${NAME}"
+```
+
+Please save these and storage credentials above in keepass (e.g. "<cluster name> azure host cluster credentials"). They will be needed in next step.
```

Comment:
Why do we need that? What uses the credentials?

Reply:
1) Storage account used to store terraform state in Azure storage (See envs.sh)
2) Service Principal used by k8s Azure cloud provider to access Azure API (in the future we will use MSI, but it's not the case in near future)

Reply:
Do we need to use the cloud provider now?

Reply:
What do you mean use cloud provider?

We are using cloud provider and we need to configure it (See [here](https://github.com/giantswarm/azure-terraform/blob/master/cloud-config/master.yaml.tmpl#L902-L916)). Only way to avoid this to use MSI, which is not the case for <1.9

JFYI: In AWS MSI-like stuff works "out-of-the-box" we just need to assign IAM policy to VM, so in AWS we don't need any AWS credentials for k8s :stuck_out_tongue_closed_eyes:

Reply:
Yes, we need that to create LB I guess.
Let's also create an follow up issue to make sure we change that.

Reply:
yep, follow up here https://github.com/giantswarm/azure-terraform/issues/5